### DISCUSSION TITLE
ADWIN ERROR

### DISCUSSION THREAD

**Discussion by ercangunbilek** [online-ml/river#1108] [General]
why do you update old versions when river is updated? My code that worked in the old version is not working now. I am getting error in ADWIN !
<img width="1048" alt="Ekran Resmi 2022-12-06 12 37 29" src="https://user-images.githubusercontent.com/33091962/205875012-bf2488f6-e35c-497c-8d02-5b7f73672f1b.png">

**Comment by ercangunbilek** [comment]
PİP İNSTALL NUMPY DOESNT WORK !

**Comment by MaxHalford** [comment]
@ercangunbilek you don't get to talk to us like that. We do our very best in our spare time to work on River. We don't work on River to deal with people's impatience.

Thank you @raphaelsty for locking the thread.
